build.gradle

```diff
@@ -0,0 +1,42 @@
+plugins {
+    id 'java'
+    id 'application'
+    id 'com.github.johnrengelman.shadow' version '7.1.2'
+}
```

Comment:
would adding the checkstyle plugin be useful to you in ensuring the code is in line with the coding standard?